Tasks/AzureKeyVaultV2/README.md

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
I'm assuming this is the same as the one from V1

Reply:
Yes, Only whats new in version 2 is added